_posts/blog/2019-08-01-lab-meeting.md

```diff
@@ -0,0 +1,58 @@
+# Whitaker Lab Meeting | 1 August 2019
```

Comment:
This file needs to be named `2019-08-01-2019-08-01-lab-meeting.md` because jekyll automatically strips the date from the start of the file name to create the blog post url but we _do_ want the date to be in that url (otherwise all lab meeting blog posts are called `lab-meeting` and only the most recent shows up.

We don't need the heading because it shows up as the title of the post, so just drop this first line too.

You _do_ need the ["front matter"](https://jekyllrb.com/docs/front-matter/) from the template and updated with this week's information. Check out `_posts\blog\template_lab-meeting.md` for the template. You shouldn't have to change too much.